r/R/dplyr.R

```diff
@@ -390,6 +409,25 @@ collect.arrow_dplyr_query <- function(x, as_data_frame = TRUE, ...) {
         tab <- RecordBatch$create(!!!cols)
       }
     }
+    # Arrange rows
+    # TODO: support sorting by expressions, not just field names
+    # TODO: support sorting by names and expressions that are valid *before* the
```

Comment:
This is solvable if you use the same machinery that we do for filter and mutate: the expressions they build always map all the way down to the underlying data.

Reply:
Ok cool, I need to look closer at that.

Reply:
Solved in 0ba20fb57c542e3099b047c792579819685c8888